evaluation.py

```diff
@@ -66,19 +66,21 @@ def stacked_predict_heatmaps(self, predict_heatmaps):
         return stacked_hourglass_heatmaps
 
     #  Saves to disk stacked predicted heatmaps and stacked ground truth heatmaps and one evaluation image
-    def save_stacked_evaluation_heatmaps(self, h, X, y, stacked_predict_heatmaps_file, stacked_ground_truth_heatmaps_file, filename):
-        predict_heatmaps=self.predict_heatmaps(h, X)
+    def save_stacked_evaluation_heatmaps(self, X, y, filename):
+        predict_heatmaps=self.predict_heatmaps(X)
         stacked_predict_heatmaps=self.stacked_predict_heatmaps(predict_heatmaps)
         stacked_ground_truth_heatmaps=self.stacked_ground_truth_heatmaps(X, y)
-
-        # Save stacked images to disk
-        plt.imsave(stacked_predict_heatmaps_file, stacked_predict_heatmaps)
-        plt.imsave(stacked_ground_truth_heatmaps_file, stacked_ground_truth_heatmaps)
+        
+        # Reshape heatmaps to 3 channels, normalize channels to [0,255]
+        stacked_predict_heatmaps = cv2.cvtColor(stacked_predict_heatmaps, cv2.COLOR_GRAY2RGB)
+        stacked_predict_heatmaps = cv2.normalize(stacked_predict_heatmaps, None, alpha=0, beta=255.0, norm_type=cv2.NORM_MINMAX, dtype=cv2.CV_32F)
+        stacked_ground_truth_heatmaps = cv2.cvtColor(stacked_ground_truth_heatmaps, cv2.COLOR_BGRA2RGB)
+        stacked_ground_truth_heatmaps = cv2.normalize(stacked_ground_truth_heatmaps, None, alpha=0, beta=255.0, norm_type=cv2.NORM_MINMAX, dtype=cv2.CV_32F)
         filename = filename
         
         heatmap_imgs = []
-        heatmap_imgs.append(cv2.imread(stacked_predict_heatmaps_file))
-        heatmap_imgs.append(cv2.imread(stacked_ground_truth_heatmaps_file))
+        heatmap_imgs.append(stacked_predict_heatmaps)
+        heatmap_imgs.append(stacked_ground_truth_heatmaps)
```

Comment:
This is great that we can store the stacked heatmaps in memory now 👍  Do we still want to write the resulting vertically stacked image to disk or should we return it to possibly just display/use in `run_evaluation.py`?

Reply:
I think to disk. Since we will wanna run this on possible many images and possibly many models. Tomorrow I will look at organizing/setting up unique fnames for these eval photos, as well as evaluating a whole batch of images at once.

Reply:
Batching would be amazing. I'm 3 hours in and halfway through. Even though I'm running on pure CPU, it's definitely not a slouch normally.